src/Microsoft.AspNet.Mvc.Localization/IHtmlLocalizer.cs

```diff
@@ -19,6 +19,9 @@ public interface IHtmlLocalizer : IStringLocalizer
         /// <returns>A culture-specific <see cref="IHtmlLocalizer"/>.</returns>
         new IHtmlLocalizer WithCulture(CultureInfo culture);
 
+        // Reviewers: I suggest the Html() methods should just return IHtmlContent because the specialized information
+        // in LocalizedHtmlString is unused. Any objections?
```

Comment:
Decided the additional cleanup I mentioned in these "Reviewers" comments can happen later. Will file an issue in the Localization repo.